Rakefile

```diff
@@ -101,10 +93,3 @@ def update_version_constant(version_file, new_version)
 
   sh('bundle', 'install', '--quiet') # To update Gemfile.lock with new wpmreleasetoolkit version
 end
-
-def check_install(version)
```

Comment:
Call site needs to also be removed on lines 69-71 otherwise `new_release` task will crash.

Reply:
Thanks! Addressed in d04fe6f73a87d002eca17a2d9d96c9cdb23628bc.